tests/e2e/cucumber/features/smoke/internalLink.feature

```diff
@@ -17,9 +17,6 @@ Feature: internal link share
     When "Alice" edits the public link named "Link" of resource "myfolder" changing role to "Invited people"
     And "Brian" opens the public link "Link"
     And "Brian" logs in from the internal link
-    
-    # comment out after fixing #9776
-    # And "Brian" opens shared-with-me page from the internal link
```

Comment:
@ScharfViktor Just fyi I removed this because with this PR, the internal link directly opens the file editor. So the following steps are actually correct.